app/commands/user/reset_account.rb

```diff
@@ -37,6 +37,8 @@ def reset_tracks!
     user.user_tracks.each do |user_track|
       UserTrack::Destroy.(user_track)
     end
+
+    Solution::RemoveUserSolutionsFromSearchIndex.defer(user)
```

Comment:
What's unclear to me is why the `UserTrack::Destroy` isn't doing this though. Do you understand that?

Reply:
I don't. It is quite weird